localflavor/tw/tw_choices.py

```diff
@@ -0,0 +1,28 @@
+#: An alphabetical list of cities and counties for use as `choices` in a formfield.
+#: Based on the administrative divisions of Taiwan.
+#: https://en.wikipedia.org/wiki/List_of_administrative_divisions_of_Taiwan
+
+TW_ADMINISTRATIVE_DIVISION_CHOICES = (
+    ("changhua_county", "彰化縣"),
```

Comment:
The displayed text should be in English and marked as translatable with gettext. We use Transifex to manage the translations - the translated version can be added to Transifix after this PR is merged. The translations are then pulled into the repo before a release, or as needed.

You can see an example of how to do this in the KW local flavor:

https://github.com/django/django-localflavor/blob/master/localflavor/kw/kw_areas.py  

I realize this is a roundabout way of getting the administrative names displayed, but this is how we allow developers to create multi-lingual applications.

Reply:
fixed, please review again.
how can i help with the translation on the Transifex?